pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
I don't think you're understandin the point that I am making. Perhaps I am not being clear.

The ISR is triggered while IFS is 1. IFS is cleared if the interrupt level becomes the inactive state. That is all fine. The ISR will repeatedly trigger while the interrupt pin is i  the active state.

However the ISR will still trigger *once* while IFS js 1 and yet the IO pin is in the *inactive* state if that state changes while not inside the ISR ar the time.